<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/target-512.png</normaloff>:/target-512.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QProgressBar" name="progressBar_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QProgressBar" name="progressBar_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QProgressBar" name="progressBar_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QProgressBar" name="progressBar_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QProgressBar" name="progressBar_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QProgressBar" name="progressBar_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 1px solid grey;
    border-radius: 5px;
	background-color: #DDDDDD;
}

QProgressBar::chunk {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 89, 14, 255), stop:1 rgba(255, 158, 49, 255));
    border-radius: 6px;
}</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/target-512.png</normaloff>:/target-512.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/gold.png&quot; width=&quot;36&quot; style=&quot;vertical-align: middle;&quot;/&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; vertical-align:sub;&quot;&gt;&lt;a style=&quot;text-decoration: none; color:#000000;&quot;  href=&quot;shop&quot;&gt;SHOP&lt;/a&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Очки: 1250</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
